gateway/enforcer/org.wso2.choreo.connect.enforcer/build.gradle

```diff
@@ -0,0 +1,98 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+description = 'WSO2 APK Enforcer'
+
+sourceSets {
+    test {
+        resources {
+            srcDirs("src/test/java")
+        }
+    }
+}
+
+dependencies {
+    compileOnly libs.tomcat
+    implementation libs.analytic.databridge
+    implementation libs.analytic.databridge.binary
+    implementation libs.analytic.publisher
+    implementation libs.andes
+    implementation libs.azure.applicationinsights
+    implementation libs.azure.messaging
+    implementation libs.azure.monitor.opentelemetry
+    implementation libs.azure.okhttp
+    implementation libs.carbon.apimgt.analytics
+    implementation libs.carbon.apimgt.gateway
+    implementation libs.carbon.apimgt.jms
+    implementation libs.commons.lang3
+    implementation libs.commons.pool
+    implementation libs.envoyproxy.controlplane
+    implementation libs.geronimo
+    implementation libs.graphql
+    implementation libs.grpc.netty
+    implementation libs.grpc.protobuf
+    implementation libs.grpc.stub
+    implementation libs.gson
+    implementation libs.guava
+    implementation libs.httpcomponents.client
+    implementation libs.jackson.annotations
+    implementation libs.jackson.databind
+    implementation libs.javax.validation.api
+    implementation libs.javax.xml.soap
+    implementation libs.json
+    implementation libs.lmax.disruptor
+    implementation libs.log4j.api
+    implementation libs.log4j.core
+    implementation libs.log4j.jcl
+    implementation libs.log4j.slf4j
+    implementation libs.minidev.json
+    implementation libs.netty.http
+    implementation libs.netty.http2
+    implementation libs.nimbus.jose.jwt
+    implementation libs.opentelemetry.api
+    implementation libs.opentelemetry.exporter.jaeger.thrift
+    implementation libs.opentelemetry.exporter.otlp
+    implementation libs.opentelemetry.exporter.zipkin
+    implementation libs.opentelemetry.extension.trace.propagators
+    implementation libs.opentelemetry.sdk
+    implementation libs.opentelemetry.semconv
+    implementation libs.snakeyaml
+    implementation libs.sun.saaj.impl
+    implementation libs.swagger.core.v3
+    implementation libs.toml
+    implementation libs.websocket
+    implementation project(':org.wso2.choreo.connect.enforcer.commons')
+    implementation project(':org.wso2.choreo.connect.discovery')
+    testImplementation libs.junit
```

Comment:
Aren't we moving to testng? We did some mock implementation work in https://github.com/binodmx/apk-test-framework and testng unit test example can be found in https://github.com/binodmx/apk-test-framework/blob/main/sample-java-unit-test/src/test/java/org/example/MyClassTest.java.

Reply:
ack. We will check on migrating to testng. It will be addressed in a new PR. Thanks for the info.